Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY . /go/src/github.com/Mirantis/k8s-AppController
 WORKDIR /go/src/github.com/Mirantis/k8s-AppController
 
 RUN apk --no-cache add git
-RUN go get ./... && go build -o kubeac cmd/kubeac/main.go && go build -o wrap cmd/wrap/main.go && go build -o bootstrap cmd/bootstrap/main.go && mv kubeac /usr/bin/kubeac && mv wrap /usr/bin/wrap && mv bootstrap /usr/bin/bootstrap && mkdir -p /opt/kubeac && mv /go/src/github.com/Mirantis/k8s-AppController/manifests /opt/kubeac/manifests && rm -fr /go
+RUN go get ./... && go build -o kubeac cmd/kubeac/main.go && go build -o wrap cmd/wrap/main.go && go build -o bootstrap cmd/bootstrap/main.go && go build -o get-status cmd/get-status/main.go && mv kubeac /usr/bin/kubeac && mv wrap /usr/bin/wrap && mv bootstrap /usr/bin/bootstrap && mv get-status /usr/bin/get-status && mkdir -p /opt/kubeac && mv /go/src/github.com/Mirantis/k8s-AppController/manifests /opt/kubeac/manifests && rm -fr /go
```

Comment:
Please add the command you are adding as a subcommand of kubeac binary to keep image size low. You can use `cmd` directory as a reference.